catalyst/README.md

```diff
@@ -153,158 +158,413 @@ whenItem(store.buttonParent)
 
 rule.subscribe(store.buttonParent)
 
-
 Or to specific function that span variants
```

Comment:
It would be better to have a complete sentence here, like

> Use rule.app for specific functions that span variants.

(I don't know if that's accurate, but the idea is to explain it more)

Reply:
fixed